src/components/footer/arrowDownSimple/arrow-down-simple.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from "./arrow-down-simple.module.scss";
+import arrowDownSimple from "../../../assets/images/arrowDownSimple.png";
+
+interface ArrowDownSimpleProps {
+  task: () => void;
+  id: string;
+}
+
+export const ArrowDownSimple: React.FC<ArrowDownSimpleProps> = ({
+  task,
+  id,
+}: Function) => {
+  return (
+    <img
+      className={styles.img}
+      src={arrowDownSimple}
+      onClick={() => task(id)}
+    ></img>
+  );
+};
```

Comment:
You will not need this component at all when you will use svg instead of png, moreover this onclick handler should be connected to <div> parent component. 

```tsx
<div onClick=(()=>{onArrowClick()})>
<ArrowDown> // imported as SVG
</div>
```

Reply:
done